CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed support for UE 4.24.
 - `MaxRPCRingBufferSize` setting has been removed. This was previously used to specify the RPC ring buffer size when generating schema. Now, `DefaultRPCRingBufferSize` is used, and can be overridden per RPC type using `RPCRingBufferSizeOverrides`.
 - `RPCRingBufferSizeMap` setting has been renamed to `RPCRingBufferSizeOverrides`.
+- Changed how FGameplayAbilitySpecHandle are generated to account for multi-worker scenarios. Handles are now only generated as a result of an ability being given to an Ability System Component (ASC), and not when FGameplayAbilitySpecHandle is constructed. Handles are now also only unique among all other handles generated by the same ASC, while in native Unreal they are unique among all handles in the game.
```

Comment:
I opted to put this under "breaking changes" since it changes semantics. This is also sort of a bug fix, not sure if we want messages relating to the same change in multiple places.